src/runtime/wasm.ts

```diff
@@ -621,7 +621,7 @@ export class WasmParticle extends Particle {
   async onHandleSync(handle: Handle, model) {
     const wasmHandle = this.handleMap.getL(handle);
     if (!model) {
-      this.exports._syncHandle(this.innerParticle, wasmHandle, 0);
+      this.exports._syncHandle(this.innerParticle, wasmHandle, null);
```

Comment:
This should be left as 0. Wasm doesn't have a concept of "null" at the type level, so we should explicitly set 0 to be interpreted as a null pointer internally.